inc/microbit/qstrdefsport.h

```diff
@@ -405,3 +405,23 @@ Q(name)
 Q(listdir)
 Q(machine)
 Q(size)
+
+Q(radio)
+Q(reset)
+Q(config)
+Q(on)
```

Comment:
This is a duplication that caused an error when compiling. If I remove this line "it just works" (tm). However, I notice that there are other duplicates (such as `Q(off)` on the following line) that have no such problems associated with them. Not sure what's going on.